src/lib/containers/download_list/DownloadConfirmation.tsx

```diff
@@ -33,37 +35,54 @@ export type DownloadConfirmationState = {
   fileCount: number
   fileSize: number
   downloadEstimate?: string
-  status: StatusEnum
   errorMessage?: string
-  ownerId?: string
 }
+
+type QueryOrFolder = {
+  queryBundleRequest?:QueryBundleRequest
+  folderId?: string
+}
+
 export type DownloadConfirmationProps = {
-  fnClose?: Function
+  fnClose?: () => void
   getLastQueryRequest?: () => QueryBundleRequest
+  folderId?: string
   topLevelControlsState?: TopLevelControlsState
   updateParentState?: <K extends keyof QueryWrapperState>(
     param: Pick<QueryWrapperState, K>,
   ) => void
   onExportTable?: () => void
+  downloadCartPageUrl?: string
 }
 
 // add files to download list
 async function addToDownload(
-  query: Query,
   token: string,
+  queryBundleRequest?: QueryBundleRequest,
+  folderId?: string,
 ): Promise<[StatusEnum, string]> {
-  const req: AddFilesToDownloadListRequest = {
-    concreteType:
-      'org.sagebionetworks.repo.model.file.AddFileToDownloadListRequest',
-    query: query,
-  }
   try {
-    const result = await SynapseClient.addFilesToDownloadList(req, token)
-    const ownerId = result.downloadList.ownerId
-    return [StatusEnum.SUCCESS, ownerId]
+    if (!SynapseClient.isInSynapseExperimentalMode()) {
```

Comment:
If not in experimental mode, do what it used to (add files from a query, or now folder, to the old Download List).